app/internal/page/fixtures_test.go

```diff
@@ -57,7 +58,7 @@ func TestPostFixturesDonorFlow(t *testing.T) {
 
 	assert.Nil(t, err)
 	assert.Equal(t, http.StatusFound, resp.StatusCode)
-	expectedPath := "/testing-start?completeLpa=1&idConfirmedAndSigned=1&lpaChecked=1&paymentComplete=1&whenCanBeUsedComplete=1&withAttorneys=1&withCP=1&withDonorDetails=1&withPeopleToNotify=1&withReplacementAttorneys=1&withRestrictions=1"
+	expectedPath := "/testing-start?completeLpa=1&idConfirmedAndSigned=1&lpaChecked=1&paymentComplete=1&peopleToNotifyCount=5&whenCanBeUsedComplete=1&withAttorneys=1&withCP=1&withDonorDetails=1&withPeopleToNotify=1&withReplacementAttorneys=1&withRestrictions=1"
```

Comment:
could have just made the `withPeopleToNotify` param be the count

Reply:
I had a really valid reason in my head for doing it this way yesterday that has now vanished. Makes sense to have one less query for sure, I'll switch it out.